### PR TITLE
Chore(doc): Update README

### PR DESCRIPTION
**Pull request by alithethird**
### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Manging Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by weiiwang01 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,42 @@
-# Pollen
+# Pollen Operator
 
-More information: https://charmhub.io/pollen
+A juju charm deploying and managing Pollen on a machine. Pollen is a scalable, high performance, free software (AGPL) web server, that provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP connections. You might think of this as ‘Entropy-as-a-Service’.
```
Comment (erinecon) [inline README.md:3]:
```suggestion
A Juju charm deploying and managing Pollen on a machine. Pollen is a scalable, high performance, free software (AGPL) web server, that provides small strings of entropy over TLS-encrypted HTTPS or clear text HTTP connections. You might think of this as ‘Entropy-as-a-Service’.
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,42 @@
...
+
+  - [Grafana Agent](https://charmhub.io/grafana-agent-k8s): Grafana Agent is an OpenTelemetry Collector distribution with configuration inspired by Terraform.
+  - [Apachi2](https://charmhub.io/apache2): The Apache HTTP Server Project is an effort to develop and maintain an open-source HTTP server for modern operating systems.
+
+For a full list of integrations, see the [Charmhub documentation](https://charmhub.io/pollen/integrations).
+
+## Learn more
+* [Read more](https://charmhub.io/pollen) <!--Link to the charm's official documentation-->
+* [Developer documentation](https://github.com/canonical/pollen) <!--Link to any developer documentation-->
+* [Official webpage](https://github.com/canonical/pollen) <!--(Optional) Link to official webpage/blog/marketing content-->
```
Comment (erinecon) [inline README.md:35]: Is the original source here? https://github.com/dustinkirkland/pollen

File: README.md [hunk 3]
```diff
@@ -1,14 +1,42 @@
...
-provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP
-connections. You might think of this as 'Entropy-as-a-Service'. This charm
-deploys a Pollen server.
+Pollinate is a free software (GPLv3) script that retrieves entropy from one or more Pollen servers and seeds the local Pseudo Random Number Generator (PRNG). You might think of this as PRNG-seeding via Entropy-as-a-Service.
 
-## Other resources
+This charm simplifies initial deployment and operations of Pollen on a machine, such as scaling the number of instances and clustering and more. As such, the charm makes it easy for those looking to take control of their own entropy generation whilst keeping operations simple.
 
-- [The Pollen source repository](https://github.com/canonical/pollen)
+For DevOps or SRE teams this charm will make operating Pollen simple and straightforward through Juju’s clean interface. It will allow easy deployment into multiple environments for testing of changes, and supports scaling out for enterprise deployments.
```
Comment (erinecon) [inline README.md:9]: Nitpick: I think this sentence can be removed

File: README.md [hunk 4]
```diff
@@ -1,14 +1,42 @@
...
+# Pollen Operator
 
-More information: https://charmhub.io/pollen
+A juju charm deploying and managing Pollen on a machine. Pollen is a scalable, high performance, free software (AGPL) web server, that provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP connections. You might think of this as ‘Entropy-as-a-Service’.
 
-Pollen is a scalable, high performance, free software (AGPL) web server, that
-provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP
-connections. You might think of this as 'Entropy-as-a-Service'. This charm
-deploys a Pollen server.
+Pollinate is a free software (GPLv3) script that retrieves entropy from one or more Pollen servers and seeds the local Pseudo Random Number Generator (PRNG). You might think of this as PRNG-seeding via Entropy-as-a-Service.
```
Comment (erinecon) [inline README.md:5]: Maybe add a link to Pollinate? Maybe https://manpages.ubuntu.com/manpages/trusty/man1/pollinate.1.html 

I don't know if this is the best link; please let me know if you know of a more appropriate one!

File: README.md (hunk 2 above)
Comment (alithethird) [reply README.md:35]: You have the original source but the Canonical source is a couple commits away and this can make differences.

Comment (erinecon) [reply README.md:34]: Ah ok, thanks for the context! No change needed for the time being

File: README.md [hunk 5]
```diff
@@ -1,14 +1,41 @@
...
+* [Read more](https://charmhub.io/pollen) <!--Link to the charm's official documentation-->
+* [Developer documentation](https://github.com/canonical/pollen) <!--Link to any developer documentation-->
+* [Official webpage](https://github.com/canonical/pollen) <!--(Optional) Link to official webpage/blog/marketing content-->
+* [Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com) <!--(Optional) Link to a page or section about troubleshooting/FAQ-->
+
+## Project and community
+* [Issues](https://github.com/canonical/pollen-operator/issues) <!--Link to GitHub issues (if applicable)-->
+* [Contributing](https://charmhub.io/pollen/docs/how-to-contribute) <!--Link to any contribution guides-->
+* [Matrix](https://matrix.to/#/#charmhub-charmdev:ubuntu.com) <!--Link to contact info (if applicable), e.g. Matrix channel-->
+`
```
Comment (erinecon) [inline README.md:41]:
```suggestion

```

**Review by erinecon — COMMENTED** [review]
The README looks great! One more nitpick and then should be good to go :)

**Review by erinecon — DISMISSED** [review]
Thanks for updating the README!

**Review by amandahla — APPROVED** [review]

**Review by weiiwang01 — APPROVED** [review]

**Review by amandahla — APPROVED** [review]

**Pull request closed without merging**